src/hooks/zkopru.ts

```diff
@@ -3,6 +3,7 @@ import { useMutation } from 'react-query'
 import { useWeb3React } from '@web3-react/core'
 import { sha512_256 } from 'js-sha512'
 import { fromWei } from '../utils/wei'
```

Comment:
corresponding utility functions in ethers.js are

fromWei => ethers.utils.formatEther
toWei => ethers.utils.parseEther